README.md

```diff
@@ -9,11 +9,253 @@ GT.M modifications - Sam Habiel.
 # Configuration Instructions
 Import all routines into Cache or GT.M. If you use Cache, make sure that the %ut namespace has been mapped to JDS.
 
-Run the following:
+Run the following to configure and start JDS for the first time, replacing 9080 with the port on which you want JDS to listen:
 
 ```
 D SETUP^VPRJCONFIG
-D GO^VPRJRCL
+D GO^VPRJRCL(9080)
 ```
 
-That's it. JDS is ready to use at this point. You can test ping by hitting `http://<ip address>:9080/ping`.
+This will set up JDS with a single listener on port 9080. From this point forward, you can start JDS with the following:
+
+```
+D START^VPRJ
+```
+
+# Testing JDS
+
+JDS provides a REST endpoint that will confirm that JDS has been successfully configured. In order to access it, launch the following URL:
+
+[http://localhost:9080/ping]
+
+If you have configured JDS to listen on a port other than 9080, modify the above URL (as well as any further example URLs in this documentation) accordingly.
+
+# JDS Documentation
+
+JDS has built-in documentation, accessible at [http://localhost:9080/documentation/index]
+
+This documentation is formatted for [API Blueprint](https://apiblueprint.org), but the API Blueprint renderers I've tried will only render the top-level documentation. I believe there may have been some sort of script or external tooling in the original JDS project for gathering and rendering the documentation into user-friendly HTML. To my knowledge, this has been lost to time.
```

Comment:
The rendering was done as part of RDK in the eHMP project. If it is important I suspect we can get the rendering part out of RDK to display this correctly.

Reply:
I have a local copy of the ehmp20 repository containing RDK, ADK, etc., but in my poking about have as yet been unable to pin down where this might be. Never used Gradle and friends, so it puts me at a distinct disadvantage :)

Reply:
I remember there being a custom theme, but that is about it.

Gradle is just a task runner (like NPM can be). It doesn't do anything magical, though understanding what it is trying to do may take a bit more.